spec/factories/questions.rb

```diff
@@ -1,7 +1,12 @@
 FactoryGirl.define do
   factory :question do
-    title "This is Question title"
-    body "This is Question body"
+    title 'This is Question title'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.